scripts/vulkan_object.py

```diff
@@ -0,0 +1,528 @@
+#!/usr/bin/env python3 -i
```

Comment:
is the idea you are using this to make sure works and once merged, try to swap out with the Vulkan-Headers version?